admin/src/utils/api.js

```diff
@@ -29,7 +29,22 @@ export const refreshCollectionCache = async (query) => {
     const response = await axios({
       url: "stellate/cache/refreshcollection",
       method: "POST",
-      data: query,
+      data: {
+        collection: query,
```

Comment:
Lets rename `query` to collection
```suggestion
        collection,
```

Reply:
updated